docs/_docs/plugins.md

```diff
@@ -722,6 +725,28 @@ The complete list of available hooks is below:
 
 ## Available Plugins
 
+### Gist
+
+The `gist` plugin is an example of tag plugin. To use it you'll need to add the
```

Comment:
yes. since it's not in the available plugin list at the bottom, I'm making it a one liner and add it there instead.